src/Kernel.php

```diff
@@ -166,28 +166,39 @@ function loadState(SnapshotStore $snapshotStore, Message $message, AggregateDefi
 function loadEvents(
     StreamName $streamName,
     ?MetadataMatcher $metadataMatcher,
-    callable $eventStoreFactory,
-    int $fromVersion = 1,
-    array $state = []
+    callable $eventStoreFactory
 ): Iterator {
     $eventStore = $eventStoreFactory();
 
     if ($eventStore->hasStream($streamName)) {
-        return $eventStore->load($streamName, $fromVersion, null, $metadataMatcher)->streamEvents();
+        return $eventStore->load($streamName, 1, null, $metadataMatcher)->streamEvents();
```

Comment:
Here is the problem (see comment above). You specify an equal match for the aggregate_version, so only one event will match, not all newer events.